apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/* global Applab */
+
+const Firebase = require("firebase");
```

Comment:
Stuff exported with `module.exports` ES5-style can be `import`ed ES6-style.  Stuff `export`ed ES6 style is harder to `require` ES5-style.

Here's the behavior as I understand it:
`import Foo from 'bar'`: If bar is using ES6 exports this sets Foo to whatever bar `export default`s. If bar is using ES5 exports it will first try to set Foo to bar's `module.exports.default`, then set it to bar's `module.exports` if not default object exists.

`import {baz} from 'bar'`: If bar is using ES6 exports this looks for an `export baz` from bar.  If it's using ES5-style exports this looks for `module.exports.baz` in bar.

@Bjvanminnen does that sound right?